frontend/src/core/editor/reducer.js

```diff
@@ -196,6 +196,7 @@ export default function reducer(
             return {
                 ...initial,
                 isRunningRequest: state.isRunningRequest,
+                selectedHelperTabIndex: state.selectedHelperTabIndex,
```

Comment:
This change/fix seems to be unrelated to changing the keyboard shortcut.

Reply:
Correct, I thought it's a small enough fix to sneak it in as a ride-a-long. :)

Reply:
My personal preference is to keep such changes aside in order to make `git blame` useful in the future. Your call nonetheless.